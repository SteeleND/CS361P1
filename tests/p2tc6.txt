14
0
2
0a0 0a1 0a2 2e1 1b1 2c2
a
abb
acc
e
abbbc
