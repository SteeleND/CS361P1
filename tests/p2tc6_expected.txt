Q = { [0] [1, 2, 0] [] [1] [1, 2] }
Sigma = { a b c }
delta =
                   a         b         c
       [0] [1, 2, 0]        []        []
 [1, 2, 0] [1, 2, 0]       [1]    [1, 2]
        []        []        []        []
       [1]        []       [1]        []
    [1, 2]        []       [1]    [1, 2]
q0 = [0]
F = { [1, 2, 0] [1] [1, 2] }

yes
yes
yes
no
no
